Taskfile.yml

```diff
@@ -170,6 +170,7 @@ tasks:
       - lint-go-consistent-integration
     cmds:
       - bin/golangci-lint run --config=.golangci-new.yml --new-from-rev=main
+#      - go vet -vettool=$(which ./bin/check-switch) ./...
```

Comment:
We should run the linter

Reply:
Let's start with better error messages :) Then please send PRs that fix those problems